packages/alfa-rules/test/sia-r19/rule.spec.tsx

```diff
@@ -0,0 +1,281 @@
+import { h } from "@siteimprove/alfa-dom/h";
+import { test } from "@siteimprove/alfa-test";
+
+import { Document } from "@siteimprove/alfa-dom";
+import { Response } from "@siteimprove/alfa-http";
+import { URL } from "@siteimprove/alfa-url";
+
+import R19, { Outcomes } from "../../src/sia-r19/rule";
+
+import { Group } from "../../src/common/group";
+
+import { evaluate } from "../common/evaluate";
+import { oracle } from "../common/oracle";
+import { passed, failed, inapplicable } from "../common/outcome";
+
+test("evaluate() passes a div element with aria-required property, whose has a valid true value", async (t) => {
+  const target = (
+    <div
+      role="textbox"
+      aria-required="true"
+      aria-label="A required textbox"
+    ></div>
```

Comment:
We can probably remove all the `aria-label`. The `aria-label` attribute accepts any string as value, so it can never fail this rule and thus is not very interesting in tests…
```suggestion
      aria-required="true"
    ></div>
```